src/searchQueryParamName 2.js

```diff
@@ -0,0 +1 @@
+export default "search";
```

Comment:
This file is already on the master branch. I think it should be removed before the merge.